### PR TITLE
Rejig Config OLEG Brand Kit

### PR DESCRIPTION
**Pull request by ethancroft**
Linked to: https://github.com/AgePartnership/web.agepartnership.co.uk/pull/2013
Ticket: https://webjobs.agepartnership.co.uk/desk/#/tickets/4962266
Task for **you** to PR: https://webjobs.agepartnership.co.uk/#tasks/13184329

I've renamed _all.scss to _config.scss
Put the _config.thing.scss classes into a config folder
Moved the utility classes into their own new file within a utilities folder
Created a _utilities.scss base file to import any _utilities.thing.scss partials

Checks:
- Best FE practice in terms of naming conventions, folder configurations
- Malicious code

### CODE REVIEW

**Comment by ethancroft** [comment]
Wahhh I'm sad I forgot to do it, espec. with me putting in Required Reading that we must do it 😢

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "oleg-brand-kit",
-  "version": "1.3.1",
+  "version": "1.3.2",
```
Comment (beccadawson) [inline package.json]: Just a patch version? 🤔

Comment (ethancroft) [reply package.json]: Hmmm good point

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "oleg-brand-kit",
-  "version": "1.3.2",
+  "version": "1.4.0",
```
Comment (beccadawson) [inline package.json]: Mmmmmmm are these not breaking changes?

**Pull request merged**